frontend/store/network.ts

```diff
@@ -38,9 +37,6 @@ export const getters = {
         }
         return pageNetwork;
     },
-    isPageNetworkFake(_state: State, getters: any) {
-        return getters.getPageNetwork === FAKE_NETWORK_NAME;
-    },
```

Comment:
Few lines down this getter is actually used, see `getters.isPageNetworkFake`

Reply:
Good catch! I removed them.